PKGBUILD

```diff
@@ -0,0 +1,61 @@
+# Script generated with create_pkgbuild.py
+# For more information: https://github.com/ros-melodic-arch/ros-build-tools-py3
+pkgdesc="ROS - Components of MoveIt! connecting to occupancy map."
+url='https://moveit.ros.org/'
+
+pkgname='ros-melodic-moveit-ros-occupancy-map-monitor'
+pkgver='1.0.3'
+arch=('any')
```

Comment:
Please use the complete arch list so that people can use it on other architectures.

See: https://github.com/ros-melodic-arch/ros-melodic-rviz/blob/master/PKGBUILD as a reference.

Reply:
Resolved in e49c6a5